### PR TITLE
Update experiment secondary name

### PR DESCRIPTION
**Pull request by julieg18**
# `main` <= #3016 <= this <= #3035

* adjusts styling and takes off brackets of commit data strings
* Updates table stories with commit message

## Demo

~https://user-images.githubusercontent.com/43496356/210012467-578968d1-42f7-496c-aca2-4dd95a226683.mov~

https://user-images.githubusercontent.com/43496356/210022501-14425aeb-e1ff-43f5-9267-8f2a813b9f14.mov

Resolves [comments](https://github.com/iterative/vscode-dvc/pull/3016#issuecomment-1367523274) in #3016

### CODE REVIEW

File: extension/src/experiments/model/collect.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ const getDisplayNameOrParent = (
     return `(${shortenForLabel(checkpointParent)})`
   }
   if (name) {
-    return `[${name}]`
```
Comment (julieg18) [inline extension/src/experiments/model/collect.ts:89]: Did we want to take off brackets in all secondary names (quick pick, tree, table) or just take off the brackets in the table? Currently this pr takes off them in all places but we could easily just do the table if preferred :)

Comment (julieg18) [inline extension/src/experiments/model/collect.ts:89]: Is there any possible issues with taking off the brackets of the experiment `displayNameOrParent` that I missed?

**Comment by julieg18** [comment]
@shcheklein, did we want to take brackets off the experiment names and the commit messages or just the commit messages?

**Comment by shcheklein** [comment]
@julieg18 we can try to keep them- hard to tell, w/o looking into the result.

Not related: why don't we see the experiments in the drop down on the demo and we see only commits ... (cc @mattseddon )?

**Comment by julieg18** [comment]
> @julieg18 we can try to keep them- hard to tell, w/o looking into the result.

Ok, will go ahead and add the brackets to the experiment names! Converting this to a draft for now! 

> Not related: why don't we see the experiments in the drop down on the demo and we see only commits ... (cc @mattseddon )?

Originally, this command ("Modify Experiment Param(s) and Resume") shows just experiment hashes, though after a discussion with Matt, we'd probably want to add the first three columns like we do with the "Select Experiments to Display in Plots" command. We can also look into adding the first three columns and the commit... Can research and update in a followup!

**Comment by shcheklein** [comment]
> Ok, will go ahead and add the brackets to the experiment names! Converting this to a draft for now!

can you do a few screenshots with and w/o them when there 10+ experiments and a few commits? that would help to decide

> Originally, this command ("Modify Experiment Param(s) and Resume") shows just experiment hashes

my question was not about additional info, sorry for the confusion ... it's more about why do we see only commits there, not experiments

**Comment by julieg18** [comment]
> can you do a few screenshots with and w/o them when there 10+ experiments and a few commits? that would help to decide

With brackets on experiment names:

image

Without brackets on experiment names:

image

**Comment by shcheklein** [comment]
I like it with brackets :) thanks @julieg18 .. wdyt?

**Comment by julieg18** [comment]
> I like it with brackets :) thanks @julieg18 .. wdyt?

Agreed, the brackets on exp names look better to me!

**Review by mattseddon — APPROVED** [review]

**Pull request merged**